pkg/revision/values.go

```diff
@@ -35,6 +36,12 @@ func ComputeValues(
 	// apply image digests from configuration, if not already set by user
 	userValues = istiovalues.ApplyDigests(version, userValues, config.Config)
 
+	// apply vendor-specific default values
+	userValues, err := istiovalues.ApplyVendorDefaults(version, userValues)
```

Comment:
If required, in a future PR, we can support this feature for IstioCNI/Ztunnel.

Reply:
I was wondering about that. The `ComputeValues` func should probably be used across all resources, right? Currently, the IstioCNI and ZTunnel controllers do this separately